src/pkg/services/m365/api/groups.go

```diff
@@ -154,6 +155,75 @@ func (c Groups) GetByID(
 	return group, nil
 }
 
+// GetAllSites gets all the sites that belong to a group. This is
+// necessary as private and shared channels gets their on individual
+// sites. All the other channels make use of the root site.
+func (c Groups) GetAllSites(
+	ctx context.Context,
+	identifier string,
+	errs *fault.Bus,
+) ([]models.Siteable, error) {
+	root, err := c.GetRootSite(ctx, identifier)
+	if err != nil {
+		return nil, clues.Wrap(err, "getting root site")
```

Comment:
The returned error already states `getting root site for group`, so this error wrap will stutter and read `getting root site: getting root site for group`.  I think either a stack is sufficient here, or (maybe preferably) the getRootSite error should change to a stack with the expectation that all callers will explain the current action.